rudder-lang/src/ast/context.rs

```diff
@@ -129,10 +135,13 @@ impl<'src> VarContext<'src> {
         name: Token<'src>,
         value: Value<'src>,
     ) -> Result<()> {
+        // println!("new_variable");
+
         if let Some(ref mut existing_kind) = self.variables.get_mut(&name) {
             match existing_kind {
                 VarKind::Generic(ref mut existing_var) => {
-                    Self::push_new_variable(name, existing_var, &value)?
+                    // println!("SOME =>>>>> NAME = {}\n VALUE = {:#?}", name, value);
```

Comment:
this should be a proper debug or trace level log

Reply:
fixed